docs/hosting.md

```diff
@@ -0,0 +1,77 @@
+<h1 align="center"><b>Hosting your Postfolio On Cloud </b></h1>
+
+## Netlify
+
+![netlify1](images/netlify.png)
```

Comment:
Rename the images properly please. put it inside ``images/hosting/``

Reply:
Okk sir